models/topics.models.js

```diff
@@ -0,0 +1,10 @@
+const db  = require('../db/connection')
+
+
+function fetchAllTopics(){
+    let queryString = "SELECT * FROM topics;"
```

Comment:
Why have we used a let here? We should always start with a const assignment and if we get to the point of needing to reassign the variable we can always change it to a let